runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -307,6 +307,16 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
 
         // endregion
 
+        /**
+         * Default client name in case the client chooses to remain anonymous.
+         */
+        String clientName = "";
+
+        /**
+         * How often should the local client checkpointer run? 0 disables it completely.
+         */
+        long checkpointTriggerFreqMillis = 0;
```

Comment:
please use Duration

Reply:
Making it Duration seems to involve a lot more changes in the nsx side of the code.  Hence naming it with 'Millis' to clarify the TimeUnit